create_aws_application

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# this is the name of the application you want to create
+NAME=$1
+# these are all the environments. the buckets will be named NAME-ENVIRONMENT
+ENVIRONMENTS=(master develop testing)
+# region where you want to create the bucket
+REGION=eu-central-1
+# aws-cli profile to use (default default)
+PROFILE=${2:-default}
+# policy name (default UserBucketFullControl)
+USERS_POLICY_NAME=${3:-UserBucketFullControl}
+
+check_jq() {
+  command -v jq >/dev/null 2>&1 || { echo "I require jq but it's not installed.  Aborting." >&2; exit 1; }
+}
+
+check_params() {
+  if [ "$NAME" = "" ]
+  then
+    echo "Usage: $0 <name of the application to create> [aws profile to use] [policy name]"
+    exit 1
+  fi
+}
+
+# create policy
+create_policy() {
+  if aws iam list-policies --scope=Local | grep -q $USERS_POLICY_NAME; then
+    echo "Found a Policy already called $USERS_POLICY_NAME. This one will be used."
+  else
+    echo "Creating a Policy"
+    aws iam create-policy --policy-name $USERS_POLICY_NAME --policy-document file://default_policy | jq '.Arn'
+  fi
+  POLICY_ARN=`aws iam list-policies --scope=Local | jq --arg pn $USERS_POLICY_NAME '.Policies[] | select(.PolicyName==$pn) | .Arn' | tr -d '"'`
+  echo $POLICY_ARN
+}
+
+create_environment() {
+  USER_NAME=$1
+  echo "user_name = $USER_NAME"
+  # creates the bucket
+  aws s3api create-bucket --bucket $1 --region $REGION --create-bucket-configuration LocationConstraint=$REGION
```

Comment:
I created another variable $BUCKET_NAME and used that. BUCKET_NAME and USER_NAME are equal